website/content/commands/artifact-list-builds.mdx

```diff
@@ -17,6 +17,8 @@ List builds.
 
 Usage: `waypoint artifact list-builds [options]`
 
+List builds.
```

Comment:
Just pointing out here that our `description` is often identical to our help text, which makes the page look a bit repetitive:

<img width="678" alt="Screen Shot 2021-08-10 at 9 51 35 AM" src="https://user-images.githubusercontent.com/8404559/128879526-e9106384-99f6-4ab5-b74d-3e7c2801fedb.png">

Not a big problem though, and I don't have any great ideas for how to make it better.

Reply:
I agree we don't need to fix this in this PR. Maybe the best fix is updating some of our help texts to be more involved than our description.

Reply:
Yeah I think I'll add a better more descriptive sentence for one of those.